nilearn/datasets/atlas.py

```diff
@@ -199,7 +196,7 @@ def fetch_atlas_craddock_2012(data_dir=None, url=None, resume=True, verbose=1):
 
 def fetch_atlas_destrieux_2009(lateralized=True, data_dir=None, url=None,
                                resume=True, verbose=1):
-    """Download and load the Destrieux cortical atlas (dated 2009) [1]_ [2]_.
+    """Download and load the Destrieux cortical atlas (dated 2009) :footcite:`Fischl2004Automatically` :footcite:`Destrieux2009sulcal`.
```

Comment:
Could you put the references on the next line for all the functions in which you added a ref? 
The first line of the docstring is used in the API page of the doc and the rendering of the citation looks weird otherwise (See comments from @tsalo in the linked issue).
Basically, do something like:

```python
def function(a, b):
    """This is a short description.
    See :footcite:`someone` for more details.
    ...
    """
```